components/sup/protocols/event.proto

```diff
@@ -98,4 +98,6 @@ message HealthCheckEvent {
   google.protobuf.StringValue stdout = 6;
   // The health check hook stderr output
   google.protobuf.StringValue stderr = 7;
+  // The heath check interval in seconds
```

Comment:
Minor - You could remove the "in seconds" now since `protobuf.Duration` has its own concept of units. I think that goes down to nanoseconds.